independent-publisher-connectors/Google BigQuery/apiProperties.json

```diff
@@ -0,0 +1,9 @@
+{
+    "properties": {
+      "connectionParameters": {},
+      "iconBrandColor": "#007ee5",

```

Comment:
@ashwanidv100 all independent publisher connector must use "#da3b01" as iconBrandColor. please follow this [template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json).

Reply:
Sure.. Will do.

Reply:
Updated